db/cache.go

```diff
@@ -1,24 +1,39 @@
 package mongik
 
 import (
+	"context"
 	"fmt"
 	"strings"
 
 	"github.com/FrosTiK-SD/mongik/constants"
-	"github.com/allegro/bigcache/v3"
+	mongik "github.com/FrosTiK-SD/mongik/models"
 )
 
 func getDBClusterFromKey(key string) string {
-	// We store DB cache in the format CLUSTER_NAME | OPERATION | QUERY
+
+	// We store DB cache in the format CLUSTER_NAME | OPERATION | QUERY | OPTIONS
 	return strings.Split(key, " | ")[0]
 }
 
-func DBCacheSet(cacheClient *bigcache.BigCache, key string, value []byte, lookupCollections ...string) error {
-	// Get the list of keys
+func DBCacheSet(mongikClient *mongik.Mongik, key string, value interface{}, lookupCollections ...string) error {
+	ctx := context.Background()
```

Comment:
No need to initialize.

Just use it directly

Reply:
resolved.